internal/installcfg/builder/builder_test.go

```diff
@@ -56,37 +56,37 @@ var _ = Describe("installcfg", func() {
 			ClusterNetworks:        []*models.ClusterNetwork{{Cidr: "1.1.1.0/24"}},
 			ServiceNetworks:        []*models.ServiceNetwork{{Cidr: "2.2.2.0/24"}},
 			MachineNetworks:        []*models.MachineNetwork{{Cidr: "1.2.3.0/24"}},
-			APIVip:                 "102.345.34.34",
-			IngressVip:             "376.5.56.6",
+			APIVip:                 "1.2.3.11",
+			IngressVip:             "1.2.3.12",
 			InstallConfigOverrides: `{"fips":true}`,
 			ImageInfo:              &models.ImageInfo{},
 			Platform:               &models.Platform{Type: common.PlatformTypePtr(models.PlatformTypeBaremetal)},
 			NetworkType:            swag.String("OpenShiftSDN"),
 		}}
 		id := strfmt.UUID(uuid.New().String())
-		// By default all the hosts in the cluster are IPv4-only
+		// By default all the hosts in the cluster are dual-stack
```

Comment:
The tests below are relying on the fact that hosts have interface in IPv6 subnet, so host with IPv4-only address cannot work in such a cluster